src/cirrus/management/deployment.py

```diff
@@ -289,7 +289,7 @@ def run_workflow(
         execution = self.get_execution_by_payload_id(wf_id)
 
         if state == "COMPLETED":
-            output = dict(ProcessPayload.from_event(json.loads(execution["output"])))
+            output = PayloadManager.from_event(json.loads(execution["output"])).payload
```

Comment:
Same here: do you need `PayloadManager`, or just `CirrusPayload`?

Reply:
I guess the `from_event` is currently specific to `PayloadManager`. I feel conflicted, because that seems like it could be just as useful on `CirrusPayload`. But if we move everything on to `CirrusPayload` then why did we separate them... 😅

So it looks like the answer to my original question is yes, you do need `PayloadManager`.